src/client/gui/lib/main.dart

```diff
@@ -21,15 +22,24 @@ import 'vm_table/vm_table_screen.dart';
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
 
-  await setupLogger();
+  // Get the current screen size
+  final screen = await getCurrentScreen();
+  Size windowSize;
 
-  await windowManager.ensureInitialized();
-  const windowOptions = WindowOptions(
+  if (screen != null && screen.frame.size.width >= 1600
+      && screen.frame.size.height >= 900) {
+    windowSize = const Size(1400, 822);  // For screens larger than 1600x900
+  } else {
+    windowSize = const Size(750, 450);   // Default window size
+  }
```

Comment:
Why is this removing the initialization of the window manager?

Also, you could make `windowSize` final and initialize it with a ternary operator instead of the `if-else`.